pkg/openapi/manager_test.go

```diff
@@ -47,15 +73,8 @@ func Test_ValidateMutationPolicy(t *testing.T) {
 		policy := v1.ClusterPolicy{}
 		_ = json.Unmarshal(tc.policy, &policy)
 
-		var errMessage string
 		err := o.ValidatePolicyMutation(&policy)
-		if err != nil {
-			errMessage = err.Error()
-		}
-
-		if errMessage != tc.errMessage {
-			t.Errorf("\nTestcase [%v] failed:\nExpected Error:  %v\nGot Error:  %v", i+1, tc.errMessage, errMessage)
-		}
+		assert.Equal(t, err == nil, tc.mustSucceed, "test #%d failed", i)
```

Comment:
didn't understand this change. in removed code, they are checking for error message but in changed code you are checking for `err == nil`